tests/factories/cht/generate.js

```diff
@@ -16,12 +27,31 @@ const PHONE_NUMBERS = [
 ];
 const PATIENT_IDS = [65421, 65422, 65423, 65424, 65425, 65426, 65427, 65428, 65429, 65430];
 
+const calculateDateOfBirth = (age) => {
+  const today = new Date();
+  const birthYear = today.getFullYear() - age;
+  const birthMonth = today.getMonth() + 1;
+  const birthDay = today.getDate();
+  return `${birthYear}-${String(birthMonth).padStart(2, '0')}-${String(birthDay).padStart(2, '0')}`;
+};
+const AGES = [25, 2, 10, 7];
+const DATE_OF_BIRTHS = AGES.map(calculateDateOfBirth);
+const date = new Date();
+date.setDate(date.getDate() - 252);
+const LAST_MENSTRUAL_PERIOD = date.toISOString().split('T')[0]; //'YYYY-MM-DD'
+
 const getReportContext = (patient, submitter) => {
+  const daysAgo = Faker.faker.number.int({ min: 1, max: 10 });
+  const currentDate = new Date();
+  currentDate.setDate(currentDate.getDate() - daysAgo);
```

Comment:
The LAST_MENSTRUAL_PERIOD is only used here, so we can move it inside this function
```suggestion
  currentDate.setDate(currentDate.getDate() - daysAgo);
  const lastMestrualPeriod = calculateLastMenstrualPeriod(new Date());
```